handlers/telegram/handler_test.go

```diff
@@ -778,136 +779,153 @@ func TestIncoming(t *testing.T) {
 
 var outgoingCases = []OutgoingTestCase{
 	{
-		Label:              "Plain Send",
-		MsgText:            "Simple Message",
-		MsgURN:             "telegram:12345",
-		MockResponseBody:   `{ "ok": true, "result": { "message_id": 133 } }`,
-		MockResponseStatus: 200,
+		Label:   "Plain Send",
+		MsgText: "Simple Message",
+		MsgURN:  "telegram:12345",
+		MockResponses: map[string][]*httpx.MockResponse{
+			"*/botauth_token/sendMessage": {
```

Comment:
I started using `*/...` url patterns where I was rewriting some tests, but other tests were still using `setSendURL` so the URL would end up being something like `http://localhost:<port>/...`... but if you find yourself rewriting ALL the tests for a handler so there no longer is any `setSendURL`.. then you use the real domain, e.g. `http://api.telegram.org/...`

Reply:
The problem here is that we mock the the URL for MO messages having attachments that is why I removed the domain so that mock service for fetching attachment can overwrite the `ApiURL` value

We can change how that works as well with a better mock later I think

Reply:
Ahh ok